BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java

```diff
@@ -426,44 +324,6 @@ private BaseBrick getFirstVisibleItem(Collection<? extends BaseBrick> items) {
         return null;
     }
 
-    /**
-     * Return all the bricks that have the tag.
-     *
-     * @param tag The tag we are looking for
-     * @return A list of all the bricks that have the same tag
-     */
-    public List<BaseBrick> getBricksByTag(@NonNull Object tag) {
```

Comment:
Are we updating a Bricks API? How could I get the bricks by tag now?

Reply:
I am moving that logic up into the app (I will post the related API on the app side once this goes in and I have artifacts)